tests/test_test_helpers.py

```diff
@@ -0,0 +1,28 @@
+from flask_common.test_helpers import (
+    Capture, DictCompare, RegexSetCompare, SetCompare
+)
+
+
+def test_set_compare():
+    assert SetCompare([1, 2, 3]) == [2, 3, 1]
+    assert not (SetCompare([1, 2, 3]) == [2, 2, 2])
+
+
+def test_regex_set_compare():
+    regex = '(.*) OR (.*) OR (.*)'
+    assert RegexSetCompare(regex, ['1', '2', '3']) == '2 OR 3 OR 1'
+    assert not (RegexSetCompare(regex, ['2', '2', '2']) == '2 OR 3 OR 1')
+
+
+def test_capture():
+	capture = Capture()
+	assert capture == 'hello'
+	assert capture() == 'hello'
+
+
+def test_dict_compare():
+	assert {'a': 'b'} == DictCompare({'a': 'b'})
+	assert not ({'a': 'c'} == DictCompare({'a': 'b'}))
+	assert {'a': 'b', 'c': 'd'} == DictCompare({'a': 'b'})
+	assert not ({'a': 'c', 'c': 'd'} == DictCompare({'a': 'b'}))
+	assert not ({'a': 'b'} == DictCompare({'c': 'd'}))
```

Comment:
(Here and in other places) This might be simpler: `assert {'a': 'b'} != DictCompare({'c': 'd'})`. Also, is there a reason why in other `*Compare` tests you have the `CompareClassOnTheLeft(...) == literal_on_the_right` but not here?

Reply:
`!=` is actually different and won't work unless we implement `__ne__`. These are only meant to be compared with `==`.